src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -296,68 +296,71 @@ public FindOccurrencesResult FindSymbolsInFile(ScriptFile scriptFile)
         {
             if (foundSymbol != null)
             {
-                int symbolOffset = referencedFiles[0].GetOffsetAtPosition(
-                    foundSymbol.ScriptRegion.StartLineNumber,
-                    foundSymbol.ScriptRegion.StartColumnNumber);
+                using (this.logger.LogExecutionTime($"Found symbol references for {foundSymbol.SymbolName} in {foundSymbol.FilePath}."))
+                {
+                    int symbolOffset = referencedFiles[0].GetOffsetAtPosition(
+                        foundSymbol.ScriptRegion.StartLineNumber,
+                        foundSymbol.ScriptRegion.StartColumnNumber);
 
-                // Make sure aliases have been loaded
-                await GetAliases();
+                    // Make sure aliases have been loaded
+                    await GetAliases();
 
-                // We want to look for references first in referenced files, hence we use ordered dictionary
-                var fileMap = new OrderedDictionary(StringComparer.OrdinalIgnoreCase);
-                foreach (ScriptFile file in referencedFiles)
-                {
-                    fileMap.Add(file.FilePath, file);
-                }
+                    // We want to look for references first in referenced files, hence we use ordered dictionary
+                    var fileMap = new OrderedDictionary(StringComparer.OrdinalIgnoreCase);
+                    foreach (ScriptFile file in referencedFiles)
+                    {
+                        fileMap.Add(file.FilePath, file);
+                    }
 
-                var allFiles = workspace.EnumeratePSFiles();
-                foreach (var file in allFiles)
-                {
-                    if (!fileMap.Contains(file))
+                    var allFiles = workspace.EnumeratePSFiles();
+                    foreach (var file in allFiles)
                     {
-                        fileMap.Add(file, new ScriptFile(file, null, this.powerShellContext.LocalPowerShellVersion.Version));
+                        if (!fileMap.Contains(file))
+                        {
+                            fileMap.Add(file, new ScriptFile(file, null, this.powerShellContext.LocalPowerShellVersion.Version));
```

Comment:
This is the one that's being taken care of in that other PR right? I got excited thinking we were about to get another speed boost ;)

Reply:
Yes. In the Diff Settings, turn on "exclude whitespace" (I think that's what it's called). That will show you what's really been changed

Reply:
Also, yeah, that change is where this one came from. So I could benchmark the change